### PR TITLE
v0.4 release announcement post

### PR DESCRIPTION
**Pull request by kkohbrok**

### CODE REVIEW

File: content/2022-02-28-v0_4-release.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+All validation steps are [indexed and documented](https://openmls.tech/book/message_validation.html), as well as thoroughly tested.
+
+## 🏗️ Still Not Finished: MLS Keeps Improving
+
+For a while now, MLS has been “close to a working group last call”, but as implementations have matured and academic research on the protocol has progressed, issues and improvement proposals have been steadily trickling in. As a result, at the time of release of OpenMLS v0.4, there is still no working group last call and no additional draft of the**#861** specification has been released. Consequently, v0.4 is [not quite feature-complete](https://github.com/openmls/openmls/issues?q=is%3Aissue+is%3Aopen+label%3A%22mls-spec+change%22), but advances the implementation to draft 12+.
```
Comment (franziskuskiefer) [inline content/2022-02-28-v0_4-release.md]: image

File: content/2022-02-28-v0_4-release.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+The MLS specification is still in motion and there are many things that need to be improved before we can move towards the next OpenMLS release. If you have a suggestion, please let us know by joining our [Zulip instance](https://openmls.zulipchat.com) or by filing an issue in our [GitHub repo](https://github.com/openmls/openmls). Or send us an email at [hello@openmls.tech](mailto:hello@openmls.tech).
+
+## 💯 Metrics
+
+Here are some data points that give an overview of our activity since the release of the roadmap in October 2021.
+
+- [Closed issues since October 2021: 1](https://github.com/openmls/openmls/issues?q=is%3Aissue+is%3Aclosed+updated%3A2021-10-01..2022-02-28+)88
+- [Merged PRs since October 2021: 2](https://github.com/openmls/openmls/pulls?q=is%3Apr+is%3Aclosed+updated%3A2021-10-01..2022-02-28+)22
+- [326 (update) commits (update link and numbers for 0.4 tag)](https://github.com/openmls/openmls/compare/b99b23c8b76c3f1bb74ec7fef931b9b6c119cd85...main)
```
Comment (franziskuskiefer) [inline content/2022-02-28-v0_4-release.md]: image
needs to get updated after the tag

Comment (kkohbrok) [reply content/2022-02-28-v0_4-release.md]: I'll just wait with merging this until the tag exists.

File: content/2022-02-28-v0_4-release.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+## 💼 Future Work
+
+The MLS specification is still in motion and there are many things that need to be improved before we can move towards the next OpenMLS release. If you have a suggestion, please let us know by joining our [Zulip instance](https://openmls.zulipchat.com) or by filing an issue in our [GitHub repo](https://github.com/openmls/openmls). Or send us an email at [hello@openmls.tech](mailto:hello@openmls.tech).
+
+## 💯 Metrics
+
+Here are some data points that give an overview of our activity since the release of the roadmap in October 2021.
+
+- [Closed issues since October 2021: 1](https://github.com/openmls/openmls/issues?q=is%3Aissue+is%3Aclosed+updated%3A2021-10-01..2022-02-28+)88
```
Comment (franziskuskiefer) [inline content/2022-02-28-v0_4-release.md]: These look a little funny. Let's putt the entire number in the links (same below).

**Review by franziskuskiefer — APPROVED** [review]
🚢

**Pull request merged**